pipeline/v3/http.py

```diff
@@ -0,0 +1,112 @@
+import io
+import typing as t
+
+import httpx
+import requests
+from requests_toolbelt import MultipartEncoder, MultipartEncoderMonitor
+from tqdm import tqdm
+
+from pipeline import current_configuration
+from pipeline.util.logging import PIPELINE_STR
+
+ACTIVE_IP = (
+    active_remote.url
+    if (active_remote := current_configuration.active_remote) is not None
+    else None
+)
+
+_client = httpx.Client(
+    base_url=ACTIVE_IP,
+    headers={
+        "Authorization": f"Bearer {current_configuration.active_remote.token}",
```

Comment:
would be nice to be able to use env vars for this stuff too, like for v2 (especially if people are using pipeline from within their own applications)